### PR TITLE
add_basic_trackpages

### PR DESCRIPTION
**Pull request by Annina2806**

### CODE REVIEW

File: frontend/src/components/tracks/index-ixd.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            Apply Now
+          </a>
+        </div>
+        <Footer addSectionClass="u-section-margin-top" />
+      </React.Fragment>
+    )
+  }
+}
+
+export default TrackPM;
```
Comment (mstockerl) [inline frontend/src/components/tracks/index-ixd.tsx]: The export has to be according to the class name in the file. So in this case `TrackIxD`. :) 

```suggestion
export default TrackIxD;
```

File: frontend/src/components/tracks/index-ixd.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+
+import Header from './header/index';
+import Footer from '../shared/footer/index';
+import TrackImage from './track-image/index';
+import WhatYouLearn from './what-you-learn/index';
+import OfferResponsibilities from './offer-responsibilities/index';
+import Expectations from './expectations/index';
+import { or, header, expectations } from './data/pm';
```
Comment (mstockerl) [inline frontend/src/components/tracks/index-ixd.tsx]: You should import the IxD data file here. ;) 

```suggestion
import { or, header, expectations } from './data/ixd';
```

File: frontend/src/components/tracks/index-ai.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            Apply Now
+          </a>
+        </div>
+        <Footer addSectionClass="u-section-margin-top" />
+      </React.Fragment>
+    )
+  }
+}
+
+export default TrackPM;
```
Comment (mstockerl) [inline frontend/src/components/tracks/index-ai.tsx]: Same as with TrackIxD 😄 

```suggestion
export default TrackAI;
```

**Comment by lustoykov** [comment]
thanks @Annina2806 for the contribution, I'll look into it in the next few days. Looks good on first sight 👍

File: frontend/src/App.tsx [hunk 4]
```diff
@@ -23,6 +23,7 @@ export class App extends React.Component {
           <Route path="/legal-disclosure" component={Legal} />
           <Route path="/privacy-policy" component={Privacy} />
           <Route path="/tracks/product-management" component={TrackPM}></Route>
+          <Route path="/tracks/product-management" component={TrackPM}></Route>
```
Comment (mstockerl) [inline frontend/src/App.tsx]: As mentioned on the phone, we need unique urls here.
This is an example for Software engineering.
```suggestion
          <Route path="/tracks/software-engineering" component={TrackSE}></Route>
```

(Don't forget to import `TrackSE` and the others at the top of the file. 🤗 )

**Pull request closed without merging**